### ISSUE TITLE
Reading EEAAQeRep.v2 does not work

### ISSUE THREAD

**Issue by hansbrenna** [metno/pyaerocom#404]
I'm trying to read EEAAQeRep.v2 and it does not work in either aerocom-updates or av-trend-patch branches. Is this a known problem @jgliss ?

```Python
import pyaerocom as pya
pya.io.ReadUngridded().read('EEAAQeRep.v2','concpm10')
```

 ```Python
92%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▎                   | 12767/13952 [24:43<02:19,  8.50it/s]---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-3-5ba5e4aa2cd8> in <module>
----> 1 test=reader.read('concpm10')

~/pyaerocom/pyaerocom/io/read_eea_aqerep_base.py in read(self, vars_to_retrieve, files, first_file, last_file, metadatafile)
    554             _file = files[i]
    555             try:
--> 556                 station_data = self.read_file(_file, var_name=var_name)
    557             except TemporalResolutionError as e:
    558                 const.print_log.warning(f'{repr(e)}. Skipping file...')

~/pyaerocom/pyaerocom/io/read_eea_aqerep_base.py in read_file(self, filename, var_name, vars_as_series)
    300                         # sometimes there's no value in the file. Set that to nan
    301                         try:
--> 302                             data_dict[header[idx]][lineidx] = np.float_(rows[idx])
    303                         except ValueError:
    304                             data_dict[header[idx]][lineidx] = np.nan

IndexError: list index out of range

```

**Comment by hansbrenna** [comment]
It fails also for concpm25 and conco3

**Comment by jgriesfeller** [comment]
Since this is my code: That looks like a corrupted file. 
The filename would be interesting. You could just skip the line by extending the except statement after line 303.

But I can check myself as soon as I have finished some other work for Michael...

**Comment by hansbrenna** [comment]
I think it's this file that causes trouble

/lustre/storeA/project/aerocom/aerocom1/AEROCOM_OBSDATA/EEA_AQeRep.v2/renamed/SE/SE_5_31069_2017_timeseries.csv'

**Comment by hansbrenna** [comment]
@jgriesfeller the file has an incomplete line at the end:

SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,10.4952231800,µg/m3,2017-06-03 03:00:00 +01:00,2017-06-03 04:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,10.8128956500,µg/m3,2017-06-03 04:00:00 +01:00,2017-06-03 05:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,10.0374536200,µg/m3,2017-06-03 05:00:00 +01:00,2017-06-03 06:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,7.5971884000,µg/m3,2017-06-03 08:00:00 +01:00,2017-06-03 09:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,12.5374159400,µg/m3,2017-06-03 09:00:00 +01:00,2017-06-03 10:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,14.9825275300,µg/m3,2017-06-03 11:00:00 +01:00,2017-06-03 12:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,16.3312971000,µg/m3,2017-06-03 12:00:00 +01:00,2017-06-03 13:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,15.4076695600,µg/m3,2017-06-03 13:00:00 +01:00,2017-06-03 14:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE0007C,STA-SE0067A,SE0067A,SPO-SE0067A_00005_100,SPP-SE0067A_00005_100_100,SPO_F-SE0067A_00005_100_100,PM10,http://dd.eionet.europa.eu/vocabulary/aq/pollutant/5,hour,15.3021942000,µg/m3,2017-06-03 14:00:00 +01:00,2017-06-03 15:00:00 +01:00,1,1
SE,SE.NV.AQ,NET-SE005

**Comment by jgriesfeller** [comment]
@hansbrenna that is an incompletely downloaded file which is hard to detect. I have deleted it and will make the reader more robust to avoid errors in the future

**Comment by jgriesfeller** [comment]
closed and replaced by #405
